src/grid.h

```diff
@@ -138,6 +138,17 @@
 #define GRID_ARROWENDS 2
 #define GRID_ARROWTYPE 3
 
+/*
+ * Helpers for unit types
+ */
+#define uValue(X) REAL(VECTOR_ELT(X, 0))[0]
+#define uData(X) VECTOR_ELT(X, 1)
+#define uUnit(X) INTEGER(VECTOR_ELT(X, 2))[0]
+#define isAbsolute(X) ((X > 1000 || (X > 100 && X < 200) || (X < 19 && X > 0 && X != 4 && X != 6)))
```

Comment:
use L_NPC instead of 0, L_NATIVE instead of 4, L_SNPC instead of 6, L_GROBX instead of 19

Reply:
If you prefer this, I'll do it. My logic was that when doing `<` and `>` it was nice to see the actual numbers